tests/distributions/test_transform.py

```diff
@@ -619,16 +631,26 @@ def test_transform_univariate_dist_logp_shape():
     assert m.logp(jacobian=True, sum=False)[0].type.shape == (4,)
 
 
-def test_univariate_transform_multivariate_dist_raises():
+@pytest.mark.parametrize(
+    argnames="jacobian", argvalues=[True, False], ids=["jacobian", "no_jacobian"]
+)
+def test_univariate_transform_multivariate_dist(jacobian):
     with pm.Model() as m:
         pm.Dirichlet("x", [1, 1, 1], transform=tr.log)
+    assert m.logp(jacobian=jacobian).type.shape == ()
 
-    for jacobian in (True, False):
-        with pytest.raises(
-            NotImplementedError,
-            match="Univariate transform LogTransform cannot be applied to multivariate",
-        ):
-            m.logp(jacobian=jacobian)
+
+@pytest.mark.parametrize(
+    argnames="jacobian", argvalues=[True, False], ids=["jacobian", "no_jacobian"]
+)
+@pytest.mark.parametrize(argnames="n", argvalues=[2, 3], ids=["n=2", "n=3"])
+def test_lkjcorr_default_transform(jacobian, n):
+    with pm.Model() as m:
+        pm.LKJCorr("Ω_triu", eta=1, n=n, transform=None)
+    assert m.compile_logp(jacobian=jacobian, sum=False)(m.initial_point())[0] == m.compile_logp(
```

Comment:
Not sure about the test.

Perhaps it's enough to test the default transform is Interval and that calling logp doesn't fail and has the right shape. This can be done in `tests/distributions/test_multivariate.py` as a specific regression test for the default transform of this RV (if this test had existed we would have had to address it when we did the breaking changes before)

Reply:
Hey! Thanks! Some questions:

> Perhaps it's enough to test the default transform is Interval 

Do you mean `m.compile_logp(jacobian=jacobian, sum=False)(m.initial_point())` is in certain interval? which one?

> calling logp doesn't fail and has the right shape.

Don't we have this already?

> This can be done in tests/distributions/test_multivariate.py as a specific regression test for the default transform of this Distribution

So you mean we move this test to a new one in `tests/distributions/test_multivariate.py`